main.py

```diff
@@ -708,7 +711,7 @@ def notify_topology_update(self):
     def notify_link_status_change(self, link):
         """Send an event to notify about a status change on a link."""
         name = 'kytos/topology.'
-        if link.is_active():
+        if link.is_active() or link.is_enabled():
```

Comment:
```suggestion
        if link.is_active() and link.is_enabled():
```
If the link is enabled but is not active, it is down, not up.

Reply:
I agree, the suggestion was accepted at commit https://github.com/kytos/topology/pull/157/commits/1a48398957307555cb4b0d5c2881106c5c3dab58